README.md

```diff
@@ -55,4 +53,19 @@ The reason you have to add this is so we can all add our own config files sepera
    Check that your local branch has the desired commits from master by doing `git log` and inspecting the history. When you are satisfied that your branch is good, do a `git push -f` to force push the new history to your remote branch. The PR should now be mergeable without any hairy merge commits.
    
 8. Some other people should come review your work and hopefully leave some suggestions/comments on what needs to be changed before a merge. Complete this work and push it to your remote branch, and the you can squash these new commits back into the old ones and `git push -f` to keep the history clean.
-9. The PR should then be merged and everybody should be happy
\ No newline at end of file
+9. The PR should then be merged and everybody should be happy
+
+
+### Story Points
+
+In agile we tend to estimate the length of time a task will take us to complete. This will allow us to better plan the amount of work we are capable of bringing in each sprint.
+
+To denote this we tend to use numbers from the Fibonacci sequence. 1, 2, 3, 5, 8, 13, 21. The logic here is that smaller tasks can accurately be estimated, however it is much harder to accurately estimate larger tasks. Hence, the larger difference between each number as it increases in size.
+
+During our first sprint, when we begin working on a ticket we should attempt to estimate the time it will take to complete. This will allow us to look back on this sprint and understand how many story points we are capable of completing. Then we can estimate future tasks before committing to completing it in a sprint.
+
+As a guideline, I think of the numbers as follows. I would encourage us to discuss and agree on this in the pull request for this Readme. Whatever we agree on, we should keep consistent across sprints, so we can properly visualise our productivity in graphs such as burndown charts.
+
```

Comment:
Feel free to share if you have a different opinion on this scale. We only have 3 sprints so it won't prove particularly essential. Alas, we must comply.

Reply:
The scaling looks pretty spot on to me, I'm all for it.

Reply:
Looks good